addons/common/functions/fnc_unGarrison.sqf

```diff
@@ -0,0 +1,45 @@
+/*
+ * Author: alganthe
+ * Used to un-garrison units garrisoned with ace_common_fnc_garrison
+ *
+ * Arguments:
+ * 0: Array of units to un-garrison <ARRAY>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Public: Yes
+ *
+*/
+#include "script_component.hpp"
+
+params [["_unitsArray", [], [[]]]];
+
+{
+    if !(isPlayer _x) then {
+        if (local _x) then {
+            _x enableAI "AUTOCOMBAT";
+            _x enableAI "PATH";
+        } else {
+            [QGVAR(enableAI), [_placedUnits, "AUTOCOMBAT"], _placedUnits] call CBA_fnc_targetEvent;
+            [QGVAR(enableAI), [_placedUnits, "PATH"], _placedUnits] call CBA_fnc_targetEvent;
```

Comment:
Not sure why this is being done inside this loop (also this variable doesn't exist).